project/src/com/google/daggerqueryui/scripts/binding_graph.js

```diff
@@ -89,22 +100,40 @@ const bindingGraph = (function() {
    *
    * @param {number} nodeId an identifier of the given node
    */
-  function getAllDeps(nodeId) {
+  function getAllEdgesToDeps(nodeId) {
     return edges.get({
       filter: edge => edge.from === nodeId
     });
   }
 
+  /**
+   * Retrieves all edges where the target node is the specified one.
```

Comment:
nit: I'm not sure I understand what this means. Would you mind rephrasing it a little? eg.

Retrieves all edges where the target node is *a dependency*.

Reply:
I mean, the node we passed to this method is equal to the destination node of the edge. I have paraphrased it [here](https://github.com/googleinterns/dagger-query/pull/47/commits/0b2d1e4ebabd8879e026e97769b41e9d8b02cd0e).